src/components/Composer.vue

```diff
@@ -6,12 +6,12 @@
 				{{ t('mail', 'From') }}
 			</label>
 			<div class="composer-fields--custom">
-				<Select
+				<NcSelect
 					id="from"
+					label="name"
 					:value="selectedAlias"
 					:options="aliases"
-					label="name"
-					track-by="selectId"
+					input-id="select-email-input"
 					:searchable="false"
 					:placeholder="t('mail', 'Select account')"
 					:clear-on-select="false"
```

Comment:
``
```suggestion
					:clear-search-on-select="false"
```
https://vue-select.org/api/props.html#clearsearchonselect